modules/client/src/test/java/org/apache/ignite/client/fakes/FakeIgnite.java

```diff
@@ -67,6 +67,17 @@ public FakeIgnite() {
      * @param name Name.
      */
     public FakeIgnite(String name) {
+        this(name, new HybridClockImpl());
+    }
+
+    /**
+     * Creates a new instance of {@link FakeIgnite} with the given {@code name} and custom {@code clock}.
+     *
+     * @param name Name.
```

Comment:
clock param is missing. Or we may remove name Name.

Reply:
Thanks, I will add this missing parameter.